client/src/routes/home/Home.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
```

Comment:
So is this Dashboard? or is this Home? @Boto1 called this Dashboard so please rename, possibly Home came back as a result of wrong merging